commander/src/bootstrapping/templates/lisk-template-ts/generators/init_generator.ts

```diff
@@ -73,4 +74,33 @@ export default class InitGenerator extends Generator {
 			{ globOptions: { dot: true, ignore: ['.DS_Store'] } },
 		);
 	}
+
+	public end(): void {
+		this.log('Generating genesis block and config.');
+		this.spawnCommandSync(`${this.destinationRoot()}/bin/run`, [
+			'genesis-block:create',
+			'--output',
+			'config/mainnet',
+		]);
+		this.spawnCommandSync(`${this.destinationRoot()}/bin/run`, [
+			'config:create',
+			'--output',
+			'config/mainnet',
+		]);
+
+		fs.mkdirSync(`${this.destinationRoot()}/.secrets/mainnet`, { recursive: true });
+
+		fs.renameSync(
+			`${this.destinationRoot()}/config/mainnet/accounts.json`,
+			`${this.destinationRoot()}/.secrets/mainnet/accounts.json`,
+		);
+
+		fs.renameSync(
+			`${this.destinationRoot()}/config/mainnet/forging_info.json`,
+			`${this.destinationRoot()}/.secrets/mainnet/forging_info.json`,
+		);
```

Comment:
Use `this.destinationPath` instead of `this.destinationRoot` with string concatenation. e.g. `this. destinationPath('bin/run')` is equal to `'${this.destinationRoot()}/bin/run'`